core/templates/pages/contributor-dashboard-page/services/contribution-and-review.service.spec.ts

```diff
@@ -28,7 +28,7 @@ require(
   'contribution-and-review.service.ts');
 
 describe('ContributionAndReviewService', function() {
-  var ContributionAndReviewService, $httpBackend, $q, UrlInterpolationService;
+  var ContributionAndReviewService, $httpBackend;
```

Comment:
If we do not need the UrlInterpolationService, please remove it from require too above.

Ditto everywhere -- remove all the unneeded requires/imports when you remove the corresponding variables.

Reply:
Done.